packages/metrics/src/features/locationWiseEventEstimations/handler.test.ts

```diff
@@ -28,6 +28,52 @@ jest.mock('../metrics/utils', () => {
   }
 })
 
+jest.mock('../../configApi', () => {
+  const originalModule = jest.requireActual('../../configApi')
+  return {
+    __esModule: true,
+    ...originalModule,
+    getApplicationConfig: () =>
+      Promise.resolve({
+        API_GATEWAY_URL: 'http://localhost:7070/',
+        CONFIG_API_URL: 'http://localhost:2021',
+        LOGIN_URL: 'http://localhost:3020',
+        AUTH_URL: 'http://localhost:4040',
+        RESOURCES_URL: 'http://localhost:3040',
+        APPLICATION_NAME: 'Farajaland CRVS',
+        CERTIFICATE_PRINT_CHARGE_FREE_PERIOD: 36500,
+        CERTIFICATE_PRINT_CHARGE_UP_LIMIT: 36500,
+        CERTIFICATE_PRINT_LOWEST_CHARGE: 0,
+        CERTIFICATE_PRINT_HIGHEST_CHARGE: 0,
+        UI_POLLING_INTERVAL: 5000,
+        FIELD_AGENT_AUDIT_LOCATIONS: 'DISTRICT',
+        APPLICATION_AUDIT_LOCATIONS: 'DISTRICT',
+        INFORMANT_MINIMUM_AGE: 16,
+        HIDE_EVENT_REGISTER_INFORMATION: false,
+        EXTERNAL_VALIDATION_WORKQUEUE: false,
+        PHONE_NUMBER_PATTERN: {
+          pattern: '/^0(7|9)[0-9]{1}[0-9]{7}$/',
+          example: '0970545855',
+          start: '0[7|9]',
+          num: '10',
+          mask: {
+            startForm: 4,
+            endBefore: 2
+          }
+        },
+        SENTRY: 'https://f892d643aab642108f44e2d1795706bc@sentry.io/1774604',
+        LOGROCKET: 'opencrvs-foundation/opencrvs-zambia',
+        NID_NUMBER_PATTERN: {
+          pattern: '/^[0-9]{9}$/',
+          example: '4837281940',
+          num: '9'
+        },
+        COUNTRY: 'zmb',
+        LANGUAGES: 'en'
+      })
+  }
+})
+
```

Comment:
Repetition

Reply:
This mock is not needed here as this is already mocked in setupJest. So i removed this.